check-services.php

```diff
@@ -64,7 +64,11 @@
 $sql = 'SELECT * FROM `devices` AS D, `services` AS S WHERE S.device_id = D.device_id ' . $where . ' ORDER by D.device_id DESC';
 
 foreach (dbFetchRows($sql) as $service) {
-    // Run the polling function
-    poll_service($service);
+    // Run the polling function only if the associated device is up
+    if ($service['status'] === "1") {
+        poll_service($service);
+    } else {
+        d_echo("\nNagios Service - ".$service['service_id']."\nSkipping service check because device ".$service['device_id']." is down.\n");
```

Comment:
I actually think we should log to eventlog as well:

`log_event("Nagios Service - {$service['service_desc']} ({$service['service_id']}) - Skipping service check because device {$service['hostname']} is down", $device, 'service', 4, $service['service_id'])`

Reply:
I think it would be too much spam in the event log in this case.

In my case, one device has 5 service checks and they are polled every 5 minutes. So eventlog would be populated with 5 new logs every 5 minutes while the device is down.

Reply:
@laf 
Just checking if you still think that we should log to the eventlog too or maybe you just missed my reply? :)